x11/urw-base35-fonts/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                urw-base35-fonts
+version             2017-07-27
+set git_version     91edd6e
+categories          x11 fonts
+maintainers         {@lemzwerg gnu.org:wl} openmaintainer
+license             AGPL-3-with-embedded-font-exception
+platforms           darwin
+supported_archs     noarch
+
+description         URW base35 fonts in various formats
+long_description    "Recent Ghostscript versions come with a new set\
+                    of URW fonts that have additional support for\
+                    the Greek and Cyrillic scripts (thus superseding\
+                    the fonts contained in the 'urw-fonts' port).\
+                    Ghostscript distributes the URW fonts in Type1\
+                    format; in addition to these files, this port\
+                    also contains the corresponding Type1 metrics\
+                    files (AFM), together with TrueType (TTF) and\
+                    OpenType (OTF) versions of the fonts."
```

Comment:
We don't normally quote the long_description. Please remove them.

Reply:
It's not possible to remove the quotes.  The reason is the `; in`, which makes Tcl interpret the `in` as a command.  Rewording the paragraph just for the sake of avoiding quotes looks silly to me (it might be bad English since it isn't my mother tongue, but that would be another issue :-)

Reply:
Got it. I didn't know that.

Reply:
Does this then end up embedding long chunks of spaces from the start of the line into the description?